dist/redhat/scylla.spec.mustache

```diff
@@ -93,6 +93,21 @@ AmbientCapabilities=CAP_SYS_NICE
 EOS
     fi
 fi
+
+# For systems with not a lot of memory, override default reservations for the slices
+# seastar has a minimum reservation of 1.5GB that kicks in, and 21GB * 0.07 = 1.5GB.
+# So for anything smaller than that we will not use percentages in the helper slice
+MEMTOTAL_BYTES=$(cat /proc/meminfo | grep MemTotal | awk '{print $2 * 1024}')
+if [ $MEMTOTAL_BYTES -lt 23008753371 ]; then
+    mkdir -p /etc/systemd/system/scylla-helper.slice.d/
+    cat << EOS > /etc/systemd/system/scylla-helper.slice.d/memory.conf
+[Slice]
+MemoryHigh=1200M
+MemoryMax=1400M
+MemoryLimit=1400M
+EOS
+fi
```

Comment:
Let's avoid this duplication.

Reply:
through a script ? Or any other idea?

Reply:
Script. In fact you need a script, to call it from install.sh. We should be thinking in terms of install.sh, with distribution packages just a way to defer the parts of install.sh that cannot be called at packaging time.

Reply:
There was already code to handle capabilities that was not shared. My new version will include a patch that comes before this one and moves that code to a shared file first. Thanks